app/app/leftSidebar/schema.js

```diff
@@ -0,0 +1,64 @@
+(function () {
+    'use strict';
+    var schema = {
+        leftSidebar: [{
```

Comment:
@zpawn Put this object to AppController, please. You can assign it right inside controller.